lib/file_database.rb

```diff
@@ -16,6 +16,19 @@ def database_empty?
     all.empty?
   end
 
+  def create(contact)
+    file_array = all.push(contact)
+
+    file.rewind
+    file.truncate(0)
+    file << file_array.to_json
+    file.flush
+  end
```

Comment:
:+1: good

the problem with opening the file everytime you call this method in the database is that you create a new file instance in memory everytime, and when the application is closed, only the last file will be closed.

it's like if you open the same file 30 times in Word